src/CMakeLists.txt

```diff
@@ -87,6 +87,16 @@ if(CELERITAS_USE_Geant4)
   add_subdirectory(accel)
 endif()
 
+if(CELERITAS_USE_VecGeom AND CELERITAS_USE_Geant4)
+  celeritas_target_include_directories(celeritas
+    PUBLIC
+    $<BUILD_INTERFACE:${Geant4_INCLUDE_DIRS}>
+  )
+  celeritas_target_link_libraries(celeritas
+    PRIVATE ${Geant4_LIBRARIES}
+  )
+endif()
+
```

Comment:
This shouldn't be necessary since we're already linking to geant4. The `INCLUDE_DIRS` is superfluous because the geant4 target automatically sets those.
```suggestion
```

Reply:
I get this build error without those lines in src/CMakeLists.txt:
```
[37/340] Building CXX object src/celeritas/CMakeFiles/celeritas_objects.dir/ext/detail/G4VecgeomConverter.cc.o
FAILED: src/celeritas/CMakeFiles/celeritas_objects.dir/ext/detail/G4VecgeomConverter.cc.o
/srv/software/gnu9/9.2.0/bin/g++ -DJSON_DIAGNOSTICS=0 -DJSON_USE_IMPLICIT_CONVERSIONS=1 -DVECCORE_ENABLE_CUDA -I/work1/celeritas/lima/work/cele/src -I/work1/celeritas/lima/work/cele/build/include -isystem /srv/software/cuda-toolkits/11.6.0/include -isystem /work1/celeritas/lima/work/cele/install/include -Wall -Wextra -pedantic -Werror -O2 -g -DNDEBUG -fPIC -fopenmp -faligned-new -std=gnu++17 -MD -MT src/celeritas/CMakeFiles/celeritas_objects.dir/ext/detail/G4VecgeomConverter.cc.o -MF src/celeritas/CMakeFiles/celeritas_objects.dir/ext/detail/G4VecgeomConverter.cc.o.d -o src/celeritas/CMakeFiles/celeritas_objects.dir/ext/detail/G4VecgeomConverter.cc.o -c /work1/celeritas/lima/work/cele/src/celeritas/ext/detail/G4VecgeomConverter.cc
In file included from /work1/celeritas/lima/work/cele/install/include/Geant4/G4RotationMatrix.hh:33,
                 from /work1/celeritas/lima/work/cele/install/include/Geant4/G4VPhysicalVolume.hh:48,
                 from /work1/celeritas/lima/work/cele/install/include/Geant4/G4PVReplica.hh:70,
                 from /work1/celeritas/lima/work/cele/src/celeritas/ext/detail/G4VecgeomConverter.hh:22,
                 from /work1/celeritas/lima/work/cele/src/celeritas/ext/detail/G4VecgeomConverter.cc:14:
/work1/celeritas/lima/work/cele/install/include/Geant4/G4ThreeVector.hh:34:10: fatal error: CLHEP/Vector/ThreeVector.h: No such file or directory
   34 | #include <CLHEP/Vector/ThreeVector.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

```

Reply:
That's weird, is CLHEP not in `/work1/celeritas/lima/work/cele/install/include`?

Reply:
Geant4 has a build option to use its own version of CLHEP, and I believe this is a well supported build mode for Geant4 (see https://geant4-data.web.cern.ch/ReleaseNotes/ReleaseNotes.11.0.html#2.).  That's why I did not have a CLHEP library locally installed.  Don't you think it is a good idea to support this installation mode as well?
Anyway, I am installing a CLHEP 2.4.5.1 locally.

Reply:
I have installed CLHEP 2.4.5.1 locally, and still the extra CMake lines are needed, otherwise the #include lines from Geant4 will not be able to find its own header files, as they don't prefix their header files with Geant4/ path.

Reply:
Damn, ok, let me see if I can reproduce this and debug what's going on.